packages/core/src/amazonqFeatureDev/controllers/chat/messenger/messenger.ts

```diff
@@ -85,25 +86,42 @@ export class Messenger {
             })
             return
         }
-
         switch (phase) {
-            case 'Approach':
-                this.dispatcher.sendErrorMessage(
-                    new ErrorMessage(
-                        ErrorMessages.tryAgain,
-                        errorMessage + messageWithConversationId(conversationId),
-                        tabID
+            case DevPhase.APPROACH:
+                if (skipTitle) {
+                    this.sendAnswer({
+                        type: 'answer',
+                        tabID: tabID,
+                        message: errorMessage,
+                    })
+                } else {
+                    this.dispatcher.sendErrorMessage(
+                        new ErrorMessage(
+                            `Sorry, we're experiencing an issue on our side. Would you like to try again?`,
+                            errorMessage + messageWithConversationId(conversationId),
+                            tabID
+                        )
                     )
-                )
+                }
+
                 break
-            case 'Codegen':
-                this.dispatcher.sendErrorMessage(
-                    new ErrorMessage(
-                        ErrorMessages.tryAgain,
-                        errorMessage + messageWithConversationId(conversationId),
-                        tabID
+            case DevPhase.CODEGEN:
+                if (skipTitle) {
+                    this.sendAnswer({
+                        type: 'answer',
+                        tabID: tabID,
+                        message: errorMessage,
+                    })
+                } else {
+                    this.dispatcher.sendErrorMessage(
+                        new ErrorMessage(
+                            `Sorry, we're experiencing an issue on our side. Would you like to try again?`,
```

Comment:
Why is there so much apologizing ("Sorry") in these messages? And why are these messages so verbose? That doesn't actually help users, it gets annoying.
```suggestion
                            `We're experiencing an issue on our side. Try again?`,
```

Reply:
Actually this is the original message we currently have, most of the scenarios this one is removed. Here we're adding just as a fallback.